cg/meta/transfer/external_data.py

```diff
@@ -144,43 +144,51 @@ def add_files_to_bundles(
+            if pass_check:
+                failed_sums.append(pass_check)
+        return failed_sums
+
+    def get_fastq_paths_to_add(
+        self, customer: str, hk_version: Version, lims_sample_id: str
+    ) -> List[Path]:
+        paths: List[Path] = self.get_all_paths(lims_sample_id=lims_sample_id, customer=customer)
+        fastq_paths_to_add: List[Path] = self.housekeeper_api.check_bundle_files(
+            file_paths=paths,
+            bundle_name=lims_sample_id,
+            last_version=hk_version,
+            tags=HK_FASTQ_TAGS,
+        )
+        return fastq_paths_to_add
+
     def add_transfer_to_housekeeper(self, ticket_id: int, dry_run: bool = False) -> None:
         """Creates sample bundles in housekeeper and adds the available files corresponding to the ticket to the
         bundle"""
-        failed_files: List[Path] = []
+        failed_paths: List[Path] = []
         cust: str = self.status_db.get_customer_id_from_ticket(ticket_id=ticket_id)
         available_samples: List[models.Sample] = self.get_available_samples(
             folder=self.get_destination_path(customer=cust), ticket_id=ticket_id
         )
-        cases_to_start = []
+        cases_to_start: list[dict] = []
         for sample in available_samples:
             cases_to_start.extend(
                 self.status_db.cases(sample_id=sample.internal_id, exclude_analysed=True)
             )
-
-        for sample in available_samples:
             lims_sample_id: str = sample.internal_id
-            paths: List[Path] = self.get_all_paths(lims_sample_id=lims_sample_id, customer=cust)
             last_version: Version = self.housekeeper_api.get_create_version(bundle=lims_sample_id)
-            fastq_paths_to_add: List[Path] = self.housekeeper_api.check_bundle_files(
-                file_paths=paths,
-                bundle_name=lims_sample_id,
-                last_version=last_version,
-                tags=HK_FASTQ_TAGS,
+            fastq_paths_to_add: List[Path] = self.get_fastq_paths_to_add(
+                customer=cust, hk_version=last_version, lims_sample_id=sample.internal_id
             )
-            failed_sums: List[Path] = []
-            for file in fastq_paths_to_add:
-                pass_check = self.check_fastq_md5sum(file)
-                if pass_check:
-                    failed_sums.append(pass_check)
-            failed_files.extend(failed_sums)
             self.add_files_to_bundles(
                 fastq_paths=fastq_paths_to_add,
                 last_version=last_version,
                 lims_sample_id=lims_sample_id,
             )
-            LOG.info("The md5 files match the md5sum for sample {}".format(lims_sample_id))
-        if failed_files:
+            failed_paths.extend(self.get_failed_samples(fastq_paths_to_add=fastq_paths_to_add))
+        if failed_paths:
             LOG.info("Changes in housekeeper will not be committed and no cases will be added")
```

Comment:
It would be great with a LOG here saying what the failed paths  were so one know what paths went wrong and can start investigating